purchase_discount/views/product_supplierinfo_view.xml

```diff
@@ -0,0 +1,23 @@
+<odoo>
+
+    <record model="ir.ui.view" id="product_supplierinfo_form_view">
+         <field name="model">product.supplierinfo</field>
+         <field name="inherit_id" ref="product.product_supplierinfo_form_view"/>
+         <field name="arch" type="xml">
+              <xpath expr="//div[field[@name='price']]" position="after">
```

Comment:
IMO here use `<field name="price" position="after">`

Reply:
I think this xpath was good as it was before this change, since it considered the price label div.
With this change, the discount label is no longer visible. I think this should be reverted.